### ISSUE TITLE
reproduce figure 5 from original ALCOVE paper

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#11]
after implementing kruschke's original loss function in #4, a useful sanity check would be to reproduce figure 5 from the original paper, which compares the results of no attentional learning rate (lr_attn = 0) vs. a small attentional learning rate (lr_attn = 0.033). the other hyperparameters you'll want to set are c = 6.5, phi = 2.0 and lr_assoc = 0.03

**Comment by alexatartaglini** [comment]
I believe I've implemented the Humble Teacher loss correctly (with maybe some optimizations possible); here is the comparison between figure 5 and my results (for lr_attn = 0 and lr_attn = 0.033 respectively): 

![figure5_vs_alcove](https://user-images.githubusercontent.com/48462071/77669508-7e924680-6f4a-11ea-86d4-21eb36707b48.jpg)

I've resized the ALCOVE plots to roughly match the figures from the Kruschke paper. For the case with no attentional learning, the results seem pretty identical, but when there is attentional learning, our program demonstrates spikes in learning types I and II later than Kruschke's. This could be an issue with my Humble Teacher loss, but I'm not sure.
